src/main/java/org/gitlab4j/api/models/Webhook.java

```diff
@@ -0,0 +1,252 @@
+package org.gitlab4j.api.models;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Webhook implements Serializable {
```

Comment:
Probably the class should be named `GroupHook` to be consistent with `ProjectHook` (and both classes could have a common ancestor, but this can be done in a latter step)

Reply:
Done :)